tests/e2e/workloads/test_data_consistency.py

```diff
@@ -155,8 +162,33 @@ def test_log_reader_writer_parallel(project, tmp_path):
                     ]
                 ),
             ]
-            run_cmd(cmd=oc_rpm_debug, timeout=600)
+            try:
+                run_cmd(cmd=oc_rpm_debug, timeout=600)
+            except Exception:
```

Comment:
are there more specific exceptions we can catch?

Reply:
No, since here we want to run debug command to fetch additional details if possible. If this fails, we just want to log the error and go on, as we are in the error handling branch already, and we will raise an exception (failing the test) or ignore the error and continue with the loop based on other factors.